### PR TITLE
Fixes #30992 - HRT API endpoint should document ipv6 parameter

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

**Review by ares — APPROVED** [review]
Thanks @stejskalleos, APJ

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @stejskalleos.

File: app/controllers/api/v2/registration_controller.rb [hunk 1]
```diff
@@ -34,7 +34,8 @@ def global
       param :name, String, required: true
       param :location_id, :number, required: true
       param :organization_id, :number, required: true
-      param :ip, String, desc: N_("not required if using a subnet with DHCP proxy")
+      param :ip, String, desc: N_("IPv4 address, not required if using a subnet with DHCP proxy")
+      param :ip6, String, desc: N_("IPv6 address, not required if using a subnet with DHCP proxy")
       param :mac, String, desc: N_("required for managed host that is bare metal, not required if it's a virtual machine")
```
Comment (ekohl) [inline app/controllers/api/v2/registration_controller.rb:39]: I didn't really read through the first iteration, but I'd drop the `not required if it's a virtual machine`. The MAC is really irrelevant when using a compute resource, but you can't do that via the registration controller.

Comment (timogoebel) [reply app/controllers/api/v2/registration_controller.rb:39]: Yeah, true. We don't to any dhcp orchestration while registering a host. The mac address is not really important.

**Pull request merged**